### PR TITLE
added new FAQ

### PR DESCRIPTION
**Pull request by emnlpstr**
added new FAQ

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

### CODE REVIEW

File: contents/faq-page/faq.yml [hunk 1]
```diff
@@ -107,7 +116,7 @@ questions:
...
       - title: É possibile accedere a PA digitale 2026 come rappresentante legale di più Scuole?
-        content: Sì. Per gestire più Scuole contemporaneamente in qualità di rappresentante legale, è necessario:<br><br>1. completare la procedura di primo accesso e attivare il profilo di una delle Scuole;<br>2. contattare l'assistenza mediante il modulo presente nella sezione "Supporto" dell'area riservata di PA digitale 2026, indicando nel messaggio i nomi e il codice IPA delle ulteriori Scuole per le quali si richiede l'attivazione del profilo.
+        content: Gentile Utente, grazie per averci contattato. Per gestire più Scuole contemporaneamente in qualità di rappresentante legale, è necessario:<br><br>1. completare la procedura di primo accesso e attivare il profilo di una delle Scuole;<br>2. contattare l'assistenza mediante il modulo presente nella sezione "Supporto" dell'area riservata di PA digitale 2026, indicando nel messaggio i nomi, gli indirizzi PEC e il codice IPA delle ulteriori Scuole per le quali si richiede l'attivazione del profilo.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:119]:
```suggestion
        content: Sì. Per gestire più Scuole contemporaneamente in qualità di rappresentante legale, è necessario:<br><br>1. completare la procedura di primo accesso e attivare il profilo di una delle Scuole;<br>2. contattare l'assistenza mediante il modulo presente nella sezione "Supporto" dell'area riservata di PA digitale 2026, indicando nel messaggio i nomi, gli indirizzi PEC e il codice IPA delle ulteriori Scuole per le quali si richiede l'attivazione del profilo.
```

**Review by bfabio — CHANGES_REQUESTED** [review]

File: contents/faq-page/faq.yml (hunk 1 above)
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: I made it

File: contents/faq-page/faq.yml [hunk 2]
```diff
@@ -440,27 +449,93 @@ questions:
...
-      - title: Come inserire l'eventuale avanzo? l’importo è messo a bilancio vincolato? 
+      - title: Come inserire l'eventuale avanzo? l’importo è messo a bilancio vincolato?
```
Comment (bfabio) [inline contents/faq-page/faq.yml:465]:
```suggestion
      - title: Come inserire l'eventuale avanzo? L’importo è messo a bilancio vincolato?
```

File: contents/faq-page/faq.yml [hunk 3]
```diff
@@ -440,27 +449,93 @@ questions:
...
+      - title: Quali enti della PA possono presentare domanda?
+        content: I soggetti attuatori, ovvero coloro che possono accedere allo strumento predisposto dall'Avviso, sono tutte le amministrazioni pubbliche, esclusi i Comuni e le Istituzioni Scolastiche.
+      - title: Quali interventi possono essere finanziati dall'Avviso della Misura 1.4.4?
+        content: "Gli interventi che possono essere finanziati sono: adesione alla piattaforma di identità digitale SPID, adesione alla piattaforma di identità digitale CIE ed erogazione di un piano formativo su disposizioni normative, linee guida e best practices in caso di integrazione a SPID e CIE con protocollo SAML2. Per ulteriori dettagli si rimanda all'Allegato 2 del presente Avviso."
+      - title: È possibile accedere al finanziamento per l'adozione di una sola delle due piattaforme di identità digitale SPID/CIE? 
+        content: Si, è possibile accedere al finanziamento previsto dall'Avviso al fine di adottare una sola delle due piattaforme, SPID o CIE, qualora una di esse sia stata già adottata dall'ente con protocollo SAML2, come specificato nell'Allegato 2.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:526]:
```suggestion
        content: Sì, è possibile accedere al finanziamento previsto dall'Avviso al fine di adottare una sola delle due piattaforme, SPID o CIE, qualora una di esse sia stata già adottata dall'ente con protocollo SAML2, come specificato nell'Allegato 2.
```

File: contents/faq-page/faq.yml [hunk 4]
```diff
@@ -440,27 +449,93 @@ questions:
...
+      - title: In quanto tempo deve essere concluso il progetto di integrazione?
+        content: Il progetto di integrazione deve essere concluso nel termine perentorio di 10 mesi dalla data di contrattualizzazione del fornitore, come specificato nell'Allegato 2.
+      - title: Quale standard di autenticazione deve applicare l'ente nell'adozione delle piattaforme di identità digitale?
+        content: OpenID Connect (OIDC) è lo standard di autenticazione attualmente utilizzato dalla quasi totalità delle moderne applicazioni web e mobile nel mondo privato ed è fortemente raccomandato, come specificato nell'Allegato 2. In alternativa è necessaria l'erogazione di un piano formativo, idoneo a costruire le basi per un successivo passaggio tecnologico.
+      - title: È possibile accedere al finanziamento in caso di progetti già avviati per integrazione SPID/CIE?
+        content: Si, è possibile accedere al finanziamento previsto dall'Avviso nel caso in cui il processo di adozione della piattaforma di identità digitale SPID/CIE sia stato avviato a decorrere dal 1° febbraio 2020 con fondi propri.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:534]:
```suggestion
        content: Sì, è possibile accedere al finanziamento previsto dall'Avviso nel caso in cui il processo di adozione della piattaforma di identità digitale SPID/CIE sia stato avviato a decorrere dal 1° febbraio 2020 con fondi propri.
```

File: contents/faq-page/faq.yml [hunk 5]
```diff
@@ -440,27 +449,93 @@ questions:
...
+  - title: 1.4.3 pagoPA Altri Enti
+    description: Le indicazioni pratiche per risolvere le problematiche tecniche ricorrenti
+    sectionId: pagopa-altri-enti
+    accordions:
+      - title: Quali enti della PA possono presentare domanda?
+        content: 'Sono invitate a presentare proposte a valere sul presente Avviso:<br>a. Le Regioni e le Province Autonome;<br>b. Le  Aziende sanitarie locali e le Aziende ospedaliere pubbliche;<br>c. Le Università e gli Istituti di Istruzione Universitaria pubblici, gli Enti e le Istituzioni di ricerca pubblici e le Istituzioni per l’Alta Formazione Artistica, Musicale e Coreutica - (AFAM) pubbliche.'
+      - title: È possibile accedere al finanziamento in caso di realizzazione del progetto prima della pubblicazione dell'Avviso "Adozione pagoPA" della Misura 1.4.3?
+        content: Si, è possibile accedere al finanziamento previsto dall'Avviso nel caso in cui l'attivazione dei servizi di incasso (pagoPA) sia stata realizzata dopo il 01 Aprile 2021 con risorse finanziarie proprie.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:480]:
```suggestion
        content: Sì, è possibile accedere al finanziamento previsto dall'Avviso nel caso in cui l'attivazione dei servizi di incasso (pagoPA) sia stata realizzata dopo il 01 Aprile 2021 con risorse finanziarie proprie.
```

File: contents/faq-page/faq.yml [hunk 6]
```diff
@@ -440,27 +449,93 @@ questions:
...
+      - title: Qual è l'obiettivo dell'Avviso "Adozione appIO" della Misura 1.4.3?
+        content: L'obiettivo del presente Avviso riferito alla Misura 1.4.3 è la migrazione e l’attivazione dei servizi digitali (appIO) seguendo una logica di “pacchetti” che identificano il numero minimo di servizi da integrare a fronte di una singola adesione all’Avviso con il fine ultimo di raggiungere una integrazione cd. full per ciascun Ente, come specificato nell'Allegato 2.
+      - title: È possibile accedere al finanziamento in caso di realizzazione del progetto prima della pubblicazione dell'Avviso "Adozione appIO" riferito della Misura 1.4.3?
+        content: Si, è possibile accedere al finanziamento previsto dall'Avviso nel caso in cui l'implementazione dei servizi digitali (appIO) sia stata realizzata a decorrere dal 01 Aprile 2021, con risorse finanziarie proprie.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:502]:
```suggestion
        content: Sì, è possibile accedere al finanziamento previsto dall'Avviso nel caso in cui l'implementazione dei servizi digitali (appIO) sia stata realizzata a decorrere dal 01 Aprile 2021, con risorse finanziarie proprie.
```

File: contents/faq-page/faq.yml [hunk 7]
```diff
@@ -440,27 +449,93 @@ questions:
...
+  - title: 1.4.4 Altri Enti
+    description: Le indicazioni pratiche per risolvere le problematiche tecniche ricorrenti
+    sectionId: altri-enti
+    accordions:
+      - title: Qual è l'importo del finanziamento concedibile?
+        content: L'importo riconoscibile per l'implementazione del pacchetto  per l'Identità digitale è pari a 14.000,00 euro e sarà erogato all'ente successivamente alla verifica da parte del Dipartimento della corretta alimentazione della Piattaforma e del raggiungimento degli obiettivi prefissati attraverso AgID e IPZS, secondo quanto previsto dall'Art. 13 dell'Avviso.
+      - title: Sono previste risorse espressamente dedicate ai territori delle regioni Abruzzo, Basilicata, Campania, Calabria, Molise, Puglia, Sardegna, Sicilia?
+        content: Si, l'Avviso destina il 40% delle risorse al finanziamento di Soggetti Attuatori ubicati nelle regioni Abruzzo, Basilicata, Campania, Calabria, Molise, Puglia, Sardegna, Sicilia, nel rispetto della priorità trasversale del PNRR relativa al “superamento dei divari territoriali” di cui all’art. 2 comma 6 bis del decreto- legge 31 maggio 2021 n.77, convertito in legge 29 luglio 2021, n. 108.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:520]:
```suggestion
        content: Sì, l'Avviso destina il 40% delle risorse al finanziamento di Soggetti Attuatori ubicati nelle regioni Abruzzo, Basilicata, Campania, Calabria, Molise, Puglia, Sardegna, Sicilia, nel rispetto della priorità trasversale del PNRR relativa al “superamento dei divari territoriali” di cui all’art. 2 comma 6 bis del decreto- legge 31 maggio 2021 n.77, convertito in legge 29 luglio 2021, n. 108. 
```

**Review by bfabio — CHANGES_REQUESTED** [review]

**Review by bfabio — APPROVED** [review]

**Pull request merged**